run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+PATH=target/debug:"$PATH" cargo run --bin foxbox "$@"
```

Comment:
That's not good enough. We need to be able to run both debug (default) and release builds. We can enforce --release to be only recognized as the first parameter if that makes things easier though.